pkg/skuba/actions/cluster/init/init_test.go

```diff
@@ -142,6 +144,68 @@ func TestInitCloudProvider(t *testing.T) {
 	}
 }
 
+func TestInitCiliumCniPlugin(t *testing.T) {
```

Comment:
Considering consolidates the two func `TestInitCiliumCniPlugin` and `TestInitUnknownCniPlugin` to one and use table-driven test cases.

Reply:
Done